Clarifications added to remote clusters, self-managed, api key based auth (#3245)
elastic/docs-content: deploy-manage/remote-clusters/remote-clusters-api-key.md

## Establish trust with a remote cluster [remote-clusters-security-api-key]

::::{note}

> If a remote cluster is part of an {{ech}} (ECH) deployment, the remote cluster server is enabled by default and it uses a publicly trusted certificate provided by the platform proxies. You can therefore skip the following steps in these instructions:

Reviewer: ```suggestion
If a remote cluster is part of an {{ech}} (ECH) deployment, the remote cluster server is enabled by default and it uses a publicly trusted certificate provided by the platform proxies. Therefore, you can skip the following steps in these instructions:
```

**On the remote (ECH) cluster:** Skip steps 1-4 (enabling the service, generating certificates, configuring SSL settings, and restarting the cluster), and go straight to step 5 (create API key step).

> **On the local (self-managed) cluster:** Do not add the `xpack.security.remote_cluster_client.ssl.certificate_authorities` setting to the configuration file because ECH uses publicly trusted certificates that don't require custom CA configuration.

Reviewer: I don't understand enough about this, but would it make sense to repeat this specific condition again in the corresponding step (step b of On the local cluster), so that it's reiterated in context? Just in case it was missed.

        xpack.security.remote_cluster_client.ssl.certificate_authorities: [ "remote-cluster-ca.crt" ]
        ```

>         ::::{tip}
>         If the remote cluster uses a publicly trusted certificate, you don't need to add the CA certificate to the `certificate_authorities` setting. This example assumes you're using the private certificates [created earlier on the remote cluster](#remote-clusters-security-api-key-remote-action), which require the CA to be added.
>         ::::
> 

Reviewer: It sounds like you don't need to add the CA certificate to the setting, but still include the setting in the file. I think we can say they can exclude this configuration setting instead?

```suggestion
        ::::{tip}
        If the remote cluster uses a publicly trusted certificate, don't include the `certificate_authorities` setting. This example assumes you're using the private certificates [created earlier on the remote cluster](#remote-clusters-security-api-key-remote-action), which require the CA to be added.
        ::::

```

2. Enter a name (*cluster alias*) for the remote cluster.
3. Specify the {{es}} endpoint URL, or the IP address or host name of the remote cluster followed by the remote cluster port (defaults to `9443`). For example, `cluster.es.eastus2.staging.azure.foundit.no:9443` or `192.168.1.1:9443`.

::::{note}

> If the remote cluster is part of an {{ech}}, {{ece}}, or {{eck}} deployment, ensure you configure the connection to use `proxy`, as the default `sniff` mode won't work in this type of environments. Refer to the [connection modes](/deploy-manage/remote-clusters/remote-clusters-self-managed.md#sniff-proxy-modes) description for more information.

Reviewer: ```suggestion
If the remote cluster is part of an {{ech}}, {{ece}}, or {{eck}} deployment, configure the connection to use `proxy`. The default `sniff` mode doesn't work in these environments. Refer to the [connection modes](/deploy-manage/remote-clusters/remote-clusters-self-managed.md#sniff-proxy-modes) description for more information.
```
